docs/patterns/components/Tag/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Demo from '../../../Demo';
+import { Tag, TagLabel } from '@deque/cauldron-react/';
+import { children, className } from '../../../props';
+
+export default class TagDemo extends Component {
+  render() {
+    return (
+      <div>
+        <Demo
+          component={Tag}
+          states={[
+            {
+              children: 'Value'
+            },
+            {
+              children: (
+                <>
```

Comment:
What's the Fragment for?

Reply:
there are 2 sibling reactNode's here: 
- `<TagLabel />`
- "value" (string)

Reply:
Ah yep, I thought it was a child of the `<TagLabel>`. My bad 👍